GraphQL/Navigating schemas, types, fetching requests, using mutations & subscriptions/proposal.md

```diff
@@ -0,0 +1,36 @@
+# TEMPLATE
+
+## :fire: Do not edit this file - copy the template and create your own file.
+
+**[Step-By-Step Technical Blog Guide](https://hq.bitproject.org/how-to-write-a-technical-blog/)**
+
+### :pushpin: Step 1
+**TITLE:**    
+Navigating schemas/types/fetching requests, using mutations/subscriptions
+
+**TOPIC:**    
+GraphQL
+
+**DESCRIPTION (5-7+ sentences):**    
+
+GraphQL, a language for APIs, was first developed by Facebook. After it was introduced, it became extremely popular as some considered it a better API than rest API because underfetching and overfetching data no longer became a problem. GraphQL allows us to specify the root field and the payload from the client side, giving greater flexibility to manipulate the data we want to request from the server. Furthermore, GraphQL has special features of allowing the client to update data, commonly known as mutations, as well as receive real-time data, the concept of subscriptions. These are the core concepts I will be covering within my blog. 
+
+### :pushpin: Step 2
+:family: **TARGET AUDIENCE (3-5+ sentences):**    
+
+My target audience are beginning coders who have little to no experience with GraphQL. I have quite a bit of experience with coding but I currently do not have much knowledge with GraphQL. My goal is to help others get to the level that I will be in after finishing this blog. All in all, this blog will be concise and easy to learn for students who have minimal experience with coding. 
```

Comment:
I think you can combine the first 2 sentences to say something like "As an experienced coder, I would like to learn more about GraphQL in the process of providing a greater understanding of GraphSQL to beginning coders". The last sentence is very similar to the first sentence, so I would suggest rephrasing it or deleting it.